app/lib/proxy_api/openscap.rb

```diff
@@ -46,5 +49,11 @@ def destroy_report(report, cname)
         false
       end
     end
+
+    private
+
+    def timeout
+      Setting[:proxy_request_timeout] > 120 ? Setting[:proxy_request_timeout] : 120
```

Comment:
I think you need something like `Setting[:proxy_request_timeout] && Setting[:proxy_request_timeout] > 120` for tests to pass.